recipes-web-service/src/main/java/com/github/mpacala00/recipeswebservice/model/UnitOfMeasure.java

```diff
@@ -0,0 +1,16 @@
+package com.github.mpacala00.recipeswebservice.model;
+
+import lombok.Builder;
+import lombok.Data;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Document("units-of-measure")
```

Comment:
https://stackoverflow.com/questions/5916080/what-are-naming-conventions-for-mongodb

user lowercase instead of snake case